### PR TITLE
All Tests Done and Complete!

### PR DESCRIPTION
**Pull request by Kayrim**
Well Done Lads!

### CODE REVIEW

**Comment by philanderson888** [comment]
Impressive work people!

Well done on your first joint project!

I am well impressed and thank you Karim for hanging everything together 'all in a day's work'!

I think you've all learned and grown immensely during this and we should press forward with this style of learning moving forwards, some teaching from me but also much lab work from you!

Have a nice evening

Phil

**Comment by Kayrim** [comment]
Someone Review this @jwKav @Hassan822

File: GroupProject01/Program.cs [hunk 1]
```diff
@@ -2,68 +2,544 @@
...
     {
-        static List<Customer> customers = new List<Customer>();
         static void Main(string[] args)
         {
+
+        }
+
+    }
+    public class CodeToTest
+    {
```
Comment (Kayrim) [inline GroupProject01/Program.cs:18]: All code to extract database data and future added code goes into this class

File: GroupProjectTest/TestClass.cs [hunk 2]
```diff
@@ -0,0 +1,217 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using GroupProject01;
+
+namespace GroupProjectTests
+{
+    class TestClass
+    {
```
Comment (Kayrim) [inline GroupProjectTest/TestClass.cs:10]: All test cases here

**Review by Stephen-Callum — COMMENTED** [review]
All good!

**Review by tim8642 — APPROVED** [review]
All good

**Comment by jwKav** [comment]
Perfect thanks a bunch Karim

**Review by jamieb97 — APPROVED** [review]
Thanks for sorting this out

**Review by jordanbenbelaid — APPROVED** [review]
Nice job guys!

**Pull request merged**